modules/core/js/src/main/scala/com/azavea/stac4s/Bbox.scala

```diff
@@ -0,0 +1,86 @@
+package com.azavea.stac4s
+
+import cats.Eq
+import cats.syntax.either._
+import cats.syntax.functor._
+import io.circe._
+import io.circe.syntax._
+
+sealed trait Bbox {
+  val xmin: Double
+  val ymin: Double
+  val xmax: Double
+  val ymax: Double
+  val toList: List[Double]
+}
```

Comment:
Hmm, the difference of the JVM Bbox and ScalaJS Bbox is only in a slingle `toExtent` _method_. Would it make sense to put into share everything that is platform independent and similar to the `toExtent` methods add via `syntax`?

Reply:
I think `Bbox` is special in that sense, everywhere else the duplication is for unlinkable dependency libraries. Do you think it's worth it for the single method?

Reply:
@jisantuc nope, I had a secret hope that this is possible